docs/vector.rst

```diff
@@ -45,6 +45,12 @@ As a temporary local variable
         # Taichi-scope
         a = ti.Vector([2, 3, 4])
 
+.. function:: ti.vec2(x, y)
```

Comment:
We probably don't want too many ways to do one thing...

Reply:
How about `ti.vec`?

Reply:
I hope we can rename `ti.Vector([xxx])`, since it's very.different from `ti.Vector(n, m)`.
One for temporary var, one for global tensor of vector.